engine/crates/composition/tests/composition/authenticated_basic/api.graphql

```diff
@@ -0,0 +1,10 @@
+type Car {
```

Comment:
why isn't `@authenticated` part of the api schema  if we show `@deprecated` and `@tag`?

Reply:
I erred towards removing anything that is federation-specific / is only meant for the engine, not for clients. But there’s really no deep rationale, let’s go with your suggestions.